content/patterns/checkbox/checkbox-pattern.html

```diff
@@ -14,64 +14,65 @@
           <li>
-            The checkbox has an accessible label provided by one of the following:
+            チェックボックスには、次のいずれかによって提供されるアクセシブルなラベルがあります。
             <ul>
-              <li>Visible text content contained within the element with role <code>checkbox</code>.</li>
-              <li>A visible label referenced by the value of <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> set on the element with role <code>checkbox</code>.</li>
-              <li><a href="#aria-label" class="property-reference">aria-label</a> set on the element with role <code>checkbox</code>.</li>
+              <li><code>checkbox</code> ロールを持つ要素内に含まれる可視テキストコンテンツ。</li>
+              <li><code>checkbox</code> ロールを持つ要素に設定された <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> の値によって参照される可視ラベル。</li>
+              <li><code>checkbox</code> ロールを持つ要素に設定された <a href="#aria-label" class="property-reference">aria-label</a>。</li>
             </ul>
           </li>
-          <li>When checked, the checkbox element has state <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>true</code>.</li>
-          <li>When not checked, it has state <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>false</code>.</li>
-          <li>When partially checked, it has state <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>mixed</code>.</li>
-          <li>If a set of checkboxes is presented as a logical group with a visible label, the checkboxes are included in an element with role <a href="#group" class="role-reference">group</a> that has the property <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> set to the ID of the element containing the label.</li>
-          <li>If the presentation includes additional descriptive static text relevant to a checkbox or checkbox group, the checkbox or checkbox group has the property <a href="#aria-describedby" class="property-reference">aria-describedby</a> set to the ID of the element containing the description.</li>
+          <li>チェックされると、チェックボックス要素には <a href="#aria-checked" class="state-reference">aria-checked</a> 状態が <code>true</code> に設定されます。</li>
+          <li>チェックされていない場合、<a href="#aria-checked" class="state-reference">aria-checked</a> 状態は <code>false</code> に設定されます。</li>
+          <li>部分的にチェックされている場合、<a href="#aria-checked" class="state-reference">aria-checked</a> 状態は <code>mixed</code> に設定されます。</li>
+          <li>チェックボックスのセットが可視ラベルを持つ論理グループとして提示される場合、チェックボックスは、<a href="#aria-labelledby" class="property-reference">aria-labelledby</a> プロパティがラベルを含む要素の ID に設定された <a href="#group" class="role-reference">group</a> ロールを持つ要素に含まれます。</li>
+          <li>プレゼンテーションにチェックボックス又はチェックボックスグループに関連する追加の説明的な静的テキストが含まれている場合、チェックボックス又はチェックボックスグループには、説明を含む要素の ID に設定された <a href="#aria-describedby" class="property-reference">aria-describedby</a> プロパティがあります。</li>
```

Comment:
presentationは「提示」とすると用語集で定まっている

```suggestion
          <li>提示にチェックボックス又はチェックボックスグループに関連する追加の説明的な静的テキストが含まれている場合、チェックボックス又はチェックボックスグループは、説明を含む要素の ID に設定された <a href="#aria-describedby" class="property-reference">aria-describedby</a> プロパティを持ちます。</li>
```

しかし提示だとしっくりこないですね……
A has a property の訳も統一したい